src/contest_effect.c

```diff
@@ -82,101 +82,101 @@ static void ContestEffect_HighlyAppealing(void)
+    eContestantStatus[eContestResources8.contestant].moreEasilyStartled = TRUE;
+    SetContestantEffectStringID(eContestResources8.contestant, CONTEST_STRING_MORE_CONSCIOUS);
 }
 
 // Makes a great appeal, but allows no more to the end.
 static void ContestEffect_GreatAppealButNoMoreMoves(void)
 {
-    sContestantStatus[shared192D0.contestant].exploded = TRUE;
-    SetContestantEffectStringID(shared192D0.contestant, CONTEST_STRING_NO_APPEAL);
+    eContestantStatus[eContestResources8.contestant].exploded = TRUE;
+    SetContestantEffectStringID(eContestResources8.contestant, CONTEST_STRING_NO_APPEAL);
 }
 
 // Can be used repeatedly without boring the JUDGE.
 static void ContestEffect_RepetitionNotBoring(void)
 {
-    sContestantStatus[shared192D0.contestant].usedRepeatableMove = TRUE;
-    sContestantStatus[shared192D0.contestant].disappointedRepeat = FALSE;
-    sContestantStatus[shared192D0.contestant].moveRepeatCount = 0;
+    eContestantStatus[eContestResources8.contestant].usedRepeatableMove = TRUE;
+    eContestantStatus[eContestResources8.contestant].disappointedRepeat = FALSE;
+    eContestantStatus[eContestResources8.contestant].moveRepeatCount = 0;
 }
 
 // Can avoid being startled by others once.
 static void ContestEffect_AvoidStartleOnce(void)
 {
-    sContestantStatus[shared192D0.contestant].jamSafetyCount = 1;
-    SetContestantEffectStringID(shared192D0.contestant, CONTEST_STRING_SETTLE_DOWN);
+    eContestantStatus[eContestResources8.contestant].jamSafetyCount = 1;
+    SetContestantEffectStringID(eContestResources8.contestant, CONTEST_STRING_SETTLE_DOWN);
 }
 
 // Can avoid being startled by others.
 static void ContestEffect_AvoidStartle(void)
 {
-    sContestantStatus[shared192D0.contestant].immune = TRUE;
-    SetContestantEffectStringID(shared192D0.contestant, CONTEST_STRING_OBLIVIOUS_TO_OTHERS);
+    eContestantStatus[eContestResources8.contestant].immune = TRUE;
+    SetContestantEffectStringID(eContestResources8.contestant, CONTEST_STRING_OBLIVIOUS_TO_OTHERS);
 }
 
 // Can avoid being startled by others a little.
 static void ContestEffect_AvoidStartleSlightly(void)
 {
-    sContestantStatus[shared192D0.contestant].jamReduction = 20;
-    SetContestantEffectStringID(shared192D0.contestant,CONTEST_STRING_LESS_AWARE);
+    eContestantStatus[eContestResources8.contestant].jamReduction = 20;
+    SetContestantEffectStringID(eContestResources8.contestant,CONTEST_STRING_LESS_AWARE);
 }
 
 // After this move, the user is less likely to be startled.
 static void ContestEffect_UserLessEasilyStartled(void)
 {
-    sContestantStatus[shared192D0.contestant].resistant = TRUE;
-    SetContestantEffectStringID(shared192D0.contestant,CONTEST_STRING_STOPPED_CARING);
+    eContestantStatus[eContestResources8.contestant].resistant = TRUE;
+    SetContestantEffectStringID(eContestResources8.contestant,CONTEST_STRING_STOPPED_CARING);
 }
 
-// Slightly startles the POK�MON in front.
+// Slightly startles the POK�MON in front.
```

Comment:
Something happened with the encoding of the `é` here.

Reply:
Can we replace these? I've actually looked at the source file with a hex editor and it's not valid UTF-8.

Reply:
Some more details:
The hex of the original comment is `50 4F 4B E9 4D 4F 4E`
50 = P
4F = O
4B = K
E9 = Start of 3-byte codepoint
4D = M
4F = O
4E = N

The correct hex for é is `C3 A9`. The confusion appears to be because the codepoint for é is 223, or 0xE9. But that by itself is not valid, as the highest-order bits would be set, indicating a multi-byte sequence.

Browsers are known to accept errors like this, and treat incorrectly-encoded UTF-8 as literal codepoint values (Also known as USC8).